src/functionalTest/resources/application-functional.yaml

```diff
@@ -1,4 +1,4 @@
-targetInstance: ${TEST_URL:http://localhost:8090}
+targetInstance: ${TEST_URL:http://rd-professional-api-pr-685.service.core-compute-preview.internal}
```

Comment:
Will need to revert this back to `localhost:8090` I believe

Reply:
Done